config/coordinator.sample.json

```diff
@@ -14,63 +14,63 @@
     "Queries": [
       {
         "Description": "Percentage of HTTP 5xx server errors",
-        "Legend": "Percentage of requests responded from the server with a 5xx status code",
+        "Legend": "Percent",
         "Query": "(sum(rate(mattermost_api_time_count{status_code=~\"5..\"}[1m]))/sum(rate(mattermost_api_time_count[1m])))*100",
         "Threshold": 0.025,
         "MinIntervalSec": 60,
         "Alert": true
       },
       {
         "Description": "Average client request duration",
-        "Legend": "Average duration of all requests as measured from the client side",
+        "Legend": "Duration in ms (avg)",
```

Comment:
You don't even need the unit because it's there in the y axis. But ok to keep this :)

Reply:
It's not, right? And it's seconds, actually! I'll fix it now
![image](https://github.com/mattermost/mattermost-load-test-ng/assets/3924815/34329d82-a265-4abe-8cdb-4be934ff13cf)

Reply:
That's because you haven't mentioned the unit in the display section. If you set that to time in seconds, then it automatically adjusts to ms when it's too low. It's far better that way.

Reply:
Yeah, I thought about that. But then we'd have to add yet another field to the coordinator metrics for users to specify the unit. I'd start with this and we can iterate later if needed.

Reply:
Why do you need user input for that? It's always in seconds right?

Reply:
This particular query is, but the queries are arbitrary and user-defined. Most of the ones we use now are plain percentages.

Reply:
Ahhh that's true. Forgot that you are templating the dashboard. All good then :+1: